src/main.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+
+from src.passwordless import PasswordlessOptions, PasswordlessApiClientBuilder, CreateAlias, \
```

Comment:
When adding multiple classes from a module, and the length goes past your max-line-length, you should format like this:

```python
from src.passwordless import (
        PasswordlessOptions,
        PasswordlessApiClientBuilder,
        CreateAlias,
)
```

Consider implementing [black](https://pypi.org/project/black/) on this repo with pre-commit-hooks (I've always used https://github.com/pre-commit/pre-commit-hooks). This will help automate and control the formatting preferences.